### PR TITLE
post JobCreated when job is created

### PR DESCRIPTION
**Pull request by stishkin**

### CODE REVIEW

**Review by Porges — APPROVED** [review]

**Review by chkeita — APPROVED** [review]

File: src/ApiService/ApiService/Functions/Jobs.cs [hunk 1]
```diff
@@ -72,6 +72,8 @@ public Async.Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymou
         if (!r.IsOk) {
             _logTracer.WithTag("HttpRequest", "POST").WithHttpStatus(r.ErrorV).Error($"failed to insert job {job.JobId:Tag:JobId}");
```
Comment (Porges) [inline src/ApiService/ApiService/Functions/Jobs.cs:73]: We should return failure when this happens
```suggestion
            _logTracer.WithTag("HttpRequest", "POST").WithHttpStatus(r.ErrorV).Error($"failed to insert job {job.JobId:Tag:JobId}");
            return r.ErrorV;
```

File: src/ApiService/ApiService/Functions/Jobs.cs [hunk 2]
```diff
@@ -72,6 +72,8 @@ public Async.Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymou
...
         }
+        await _context.Events.SendEvent(new EventJobCreated(job.JobId, job.Config, job.UserInfo));
```
Comment (chkeita) [inline src/ApiService/ApiService/Functions/Jobs.cs:75]: this should be sent only if the job was created

File: src/ApiService/ApiService/Functions/Jobs.cs [hunk 3]
```diff
@@ -71,7 +71,16 @@ public Async.Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymou
         var r = await _context.JobOperations.Insert(job);
         if (!r.IsOk) {
             _logTracer.WithTag("HttpRequest", "POST").WithHttpStatus(r.ErrorV).Error($"failed to insert job {job.JobId:Tag:JobId}");
+            return await _context.RequestHandling.NotOk(
+                req,
+                new Error(
```
Comment (stishkin) [inline src/ApiService/ApiService/Functions/Jobs.cs:76]: @Porges  you mean like that ?
@chkeita  - yes it fails

Comment (Porges) [reply src/ApiService/ApiService/Functions/Jobs.cs:76]: @stishkin I think probably just `UNABLE_TO_CREATE`; there is no container setup here?

Comment (stishkin) [reply src/ApiService/ApiService/Functions/Jobs.cs:76]: @chkeita  - what do you mean ? job created should be sent when it failed to insert ?

Comment (chkeita) [reply src/ApiService/ApiService/Functions/Jobs.cs:76]: oh never mind i misunderstood the call

**Pull request merged**